app/src/androidTest/java/ch/epfl/qedit/util/LocationHelper.java

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.qedit.util;
+
+import android.content.Context;
+import android.location.Criteria;
+import android.location.LocationManager;
+import ch.epfl.qedit.backend.location.MockLocService;
+
+public class LocationHelper {
+    public static void initLocationProvider(Context context) {
```

Comment:
I don't understand what this class is for. Please add more comments !

Reply:
Okay, you are right.